com.coeck_studios.Dinonuggys-Journey.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, this should be maintained upstream.

Reply:
[Done](https://codeberg.org/CoEck/Dinonuggys-Journey/src/branch/main/com.coeck_studios.Dinonuggys-Journey.appdata.xml)